ducktape/command_line/main.py

```diff
@@ -109,7 +127,14 @@ def main():
 
     # Initializing the cluster is slow, so do so only if
     # tests are sure to be run
-    session_context.cluster = VagrantCluster()
+    try:
+        (cluster_mod_name, cluster_class_name) = args.cluster.rsplit('.', 1)
+        cluster_mod = importlib.import_module(cluster_mod_name)
+        cluster_class = getattr(cluster_mod, cluster_class_name)
+        session_context.cluster = cluster_class()
```

Comment:
do we want the cluster to be a subclass of Cluster? 

I guess the test runner will still (probably) fail if it doesn't walk and talk like a Cluster, but fail-fast and all that jazz.